src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -172,6 +194,35 @@ public FormValidation doCheckRegion(@QueryParameter String region){
         }
         return ret;
     }
-    
-    
+
+    public FormValidation doValidateS3BucketConfig(@QueryParameter String container, @QueryParameter String prefix,
+                                                   @QueryParameter String region){
+        FormValidation ret = FormValidation.ok("success");
```

Comment:
You must check some permission here, probably `ADMINISTER`, and it should also be `@RequirePOST` to make sure there is no possibility of a SSRF or CSRF attack.